rlclientlib/logger/event_queue.h

```diff
@@ -29,6 +32,7 @@ namespace reinforcement_learning {
       if (!_queue.empty())
       {
         *item = std::move(_queue.front());
+        _capacity = (std::max)(0, static_cast<int>(_capacity) - static_cast<int>(item->size()));
```

Comment:
Why put function name in brackets?

Reply:
Workaround for disambiguation with windows.h macros.